modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure demonstrates how to deploy application workloads in separate namespaces.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Add the `istio-discovery=mesh-1` label to ensure that the namespace is discovered by the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio-discovery=mesh-1
+----
+
+. Add the `istio.io/rev=mesh-1` label to the namespace to enable sidecar injection into all the pods by default while ensuring that pods in this namespace are mapped to the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio.io/rev=mesh-1
+----
++
+[NOTE]
+====
+You can verify the `mesh-1` revision name by running the following command:
+[source,terminal]
+----
+$ oc get istiorevisions
+----
+====
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -n app-ns-1 \
+   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml
```

Comment:
I tried it, and it looks like the command is missing a "\" at the end, as only the first service was created. Adding this fixed it.
```suggestion
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml \
```